OpenKh.Command.PmoConverter/OpenKh.Command.PmoConverter.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+      <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.1.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+      <ProjectReference Include="..\OpenKh.Common\OpenKh.Common.csproj" />
+      <ProjectReference Include="..\OpenKh.Bbs\OpenKh.Bbs.csproj" />
+      <ProjectReference Include="..\OpenKh.Engine\OpenKh.Engine.csproj" />
+      <ProjectReference Include="..\OpenKh.Engine.MonoGame\OpenKh.Engine.MonoGame.csproj" />
+      <ProjectReference Include="..\OpenKh.Game\OpenKh.Game.csproj" />
+      <ProjectReference Include="..\OpenKh.WinShell.IMDUtilities\OpenKh.WinShell.IMDUtilities.csproj" />
```

Comment:
Oh yeah, I included it because I wanted to use the `MarshallBitmap` from it because I didn't know we had a `Png` class.
I'll remove those references in a second.

Reply:
Thanks for noticing and changing this on such short notice 👍